hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/hbase/TestSparkHoodieHBaseIndex.java

```diff
@@ -16,15 +16,17 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.client.functional;
+package org.apache.hudi.index.hbase;
```

Comment:
moved to the same package as `SparkHoodieHBaseIndex` so that some methods with RDD in the signature can be changed to package access